recipes/slack.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/* (c) Tomas Majer <tomasmajer@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Notify Slack of successful deployment
+ */
+task('deploy:slack', function () {
+    global $php_errormsg;
```

Comment:
Understand. This slack notifier is basically the same as newrelic and have same issues as newrelic. Let me know if there is any better way how to display warning/error from this task and not break whole deployment.